testsuite/features/step_definitions/command_steps.rb

```diff
@@ -367,6 +367,8 @@
     pid = process.split(' ')[0]
     $server.run("kill #{pid}", check_errors: false)
     log "Reposync of channel #{channel} killed"
+
+    raise 'After 7200s we still have spacewalk-repo-sync processes running' if reposync_left_running_streak > 7200
```

Comment:
raise "After 7200s we still have a spacewalk-repo-sync process running:\n#{process}" if reposync_left_running_streak > 7200

Reply:
Why we need to repeat information?
We are already printing `log "Reposync of channel #{channel} left running"` in the console output line before that one. It will not give us much more details printing `process`

Reply:
OK

then just the text part (English nitpicks)
```
raise 'We have a reposync process still running after 2 hours' if reposync_left_running_streak > 7200
```

Reply:
Ok, I wanted to match the number with the text to simplify reading code, but ok, log reading will be easier with 2h.

Reply:
But the text "We have a reposync process still running after 2 hours" is not totally clear, as what it took 2 hours is the total time waiting for all reposyncs.

Reply:
No.

As already explained, it is the time for THIS reposync.